### ISSUE TITLE
Notification: adding to conda-forge

### ISSUE THREAD

**Issue by jfy133** [hsolbrig/rdflib-shim#6]
I would like to let you know I am adding rdflib-shim to the conda-forge code distribution platform (in parallel to your existing pypi package), in the context of the LinkML project.

For your reference, the conda-forge PR is here: https://github.com/conda-forge/staged-recipes/pull/26815

**Comment by jfy133** [comment]
This is now merged into conda-forge
